spec.bs

```diff
@@ -2792,6 +2836,22 @@ modified such that the first step of the algorithm that runs when a user-agent c
 [=traversable navigable=]'s [=system visibility state=] calls the [=Document/inclusive descendant
 navigables=] algorithm with [=inclusive-dn-unfenced|unfenced=] set to true.
 
+<h3 id=events>Events</h3>
+
+The {{Fence/notifyEvent}} method of the {{Fence}} interface introduces a new event type, `fencedtreeclick`.
+
+The [=event handlers on elements, Document objects, and Window objects=] section of [[HTML]] is modified to 
+include a new row. The [=event handler=] name is `onfencedtreeclick`, and the [=event handler event type=] is
```

Comment:
Sometimes you use backticks and other times you use double quotes when wrapping `fencedtreeclick`. I think both are wrong though. There should be an actual definition for this wrapped in code, like https://html.spec.whatwg.org/multipage/media.html#event-media-change, and each reference of it should be a code-wrapped reference to that definition.

For examples in our own spec, see https://wicg.github.io/fenced-frame/#automatic-beacon-event-type-reservedtop_navigation and https://wicg.github.io/fenced-frame/#ref-for-automatic-beacon-event-type-reservedtop_navigation%E2%91%A1.

Reply:
Yeah, see https://github.com/WICG/fenced-frame/pull/156#discussion_r1586865226.